query-parser/tests/find_tests.rs

```diff
@@ -208,3 +210,27 @@ fn can_parse_simple_or_and_join() {
                    )),
                ]);
 }
+
+#[test]
+fn can_parse_order_by() {
+    let invalid = "[:find ?x :where [?x :foo/baz ?y] :order]";
```

Comment:
Can you assert that repeating an ordered variable fails?  (Or, if you want to file follow-up, test that it _doesn't_ fail and that the later direction takes precedent?)